Tests/SentryTests/Integrations/Performance/SentryPerformanceTracker+Testing.h

```diff
@@ -1,6 +1,7 @@
 #import "SentryPerformanceTracker.h"
 #import "SentrySpan.h"
 #import "SentrySpanId.h"
+#import "SentryTracer.h"
```

Comment:
This was implicitly imported from... somewhere? This file tries to use `SentryTracerDelegate`, and this wound up breaking after I moved some stuff around.